src/test/java/YakuninaTest.java

```diff
@@ -1,35 +1,34 @@
-import org.openqa.selenium.chrome.ChromeDriver;
 import org.testng.annotations.Test;
 import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
+import runner.BaseTest;
+
 import static org.testng.Assert.assertEquals;
 import java.time.Duration;
 
-public class YakuninaTest {
-    @Test
-    public void ProjectTest() {
-
-        WebDriver driver = new ChromeDriver();
-        driver.get("https://www.selenium.dev/selenium/web/web-form.html");
-
-        String title = driver.getTitle();
-        assertEquals("Web form", title);
-
-        driver.manage().timeouts().implicitlyWait(Duration.ofMillis(500));
-
-        WebElement textBox = driver.findElement(By.name("my-text"));
-        WebElement submitButton = driver.findElement(By.cssSelector("button"));
+public class YakuninaTest extends BaseTest {
 
-        textBox.sendKeys("Selenium");
-        submitButton.click();
-
-        WebElement message = driver.findElement(By.id("message"));
-        String value = message.getText();
-        assertEquals("Received!", value);
-
-        driver.quit();
+    //TC_004
+    @Test
+    public void testAccountButtonOpensLoginPage() {
+        String title = getDriver().getTitle();
+        assertEquals(title, "AskOmDch – Become a Selenium automation expert!");
+        getDriver().manage().timeouts().implicitlyWait(Duration.ofMillis(500));
```

Comment:
We already have in a baseTest an implicit wait. You probably can skip that line.

Reply:
thank you for your comment! I'll fix that in my next PR